MapboxCoreNavigation/RouteController.swift

```diff
@@ -146,6 +146,18 @@ public protocol RouteControllerDelegate: class {
      */
     @objc(routeController:shouldPreventReroutesWhenArrivingAtWaypoint:)
     optional func routeController(_ routeController: RouteController, shouldPreventReroutesWhenArrivingAt waypoint: Waypoint) -> Bool
+    
+    
+    /**
+     Called when the route controller will disable battery monitoring.
+     
+     Implementing this method will allow developers to change whether battery monitoring is disabled when `RouteController` is deinited.
+     
+     - parameter routeController: The route controller that will change the state of battery monitoring.
+     - returns: A bool representing the value you would like `UIDevice.isBatteryMonitoringEnabled` set to.
```

Comment:
The return value here indicates whether to disable battery monitoring when the RouteController is deinited, not the value to set it to. If this method is not implemented, battery monitoring will be disabled when navigation ends.

With that doc change (or something close to it) I think we're good to merge this.